docs/user-guide/glossary.md

```diff
@@ -77,6 +77,13 @@ A storage system for electrical energy.
 A generator that produces combined heat and power ([CHP](#chp)). Usually
 powered via combustion of some form of fuel.
 
+#### Consumer
+
+This aggregates all remaining electrical energy [consumption](#consumption) in a microgrid that are not covered by other asset types and components.
```

Comment:
Maybe we want to mention that when used with a metric, like consumer power or consumer power, it has this meaning.  Just in that context.